src/Upgrade/Application/Dto/StepsResponseDto.php

```diff
@@ -26,10 +26,15 @@ class StepsResponseDto extends ResponseDto
      */
     protected ?ComposerLockDiffDto $composerLockDiffDto = null;
 
+    /**
+     * @var \Upgrade\Application\Dto\IntegratorResponseDto|null
+     */
+    protected ?IntegratorResponseDto $integratorResponseDto = null;
+
     /**
      * @var string|null
      */
-    protected ?string $pullRequestWarning = null;
+    protected ?string $majorAvailableInfo = null;
```

Comment:
I'd suggest using an empty string as a default value and making it `string` only will simplify the code and make it cleaner.

Reply:
++